pkg/metrics/metrics.go

```diff
@@ -208,6 +184,37 @@ func (pc *PrometheusNodeConfig) Configure() error {
 	return nil
 }
 
+// retrieveMetricsEndpoint returns an existing Endpoint object or creates a new one in the operator namespace
+// if it does not already exist.
+func (pc *PrometheusNodeConfig) retrieveMetricsEndpoint() (*v1.Endpoints, error) {
+	existingEndpoint, err := pc.k8sclientset.CoreV1().Endpoints(pc.namespace).Get(context.TODO(), windowsMetricsResource, metav1.GetOptions{})
+	if err == nil {
+		return existingEndpoint, nil
+	}
+	// create new Endpoint
+	newEndpoint := &v1.Endpoints{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Endpoints",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      windowsMetricsResource,
+			Namespace: pc.namespace,
+			Labels:    map[string]string{"app.kubernetes.io/name": windowsMetricsResource},
```

Comment:
this label needs to be changed to `name`

Reply:
missed this change, thanks for pointing out!